themes/theme-ios11/pages/Search/components/Bar/index.jsx

```diff
@@ -19,12 +20,13 @@ const viewMap = {
  */
 const SearchBar = () => (
   <Consume context={RouteContext} props={routeMap}>
-    {({ searchPhrase, filters }) => (
+    {({ searchPhrase, filters, pattern }) => (
       <Consume context={ViewContext} props={viewMap}>
         {({ ref, setTop }) => (
           <FilterBar
             searchPhrase={searchPhrase}
             filters={filters}
```

Comment:
i'd prefer to remove the category filter from the `filters` before the  object is passed down to the `FilterBar`. then you could remove a lot of logic related the CATEGORY_ALL route from the underlying components.